src/lib.rs

```diff
@@ -781,12 +627,10 @@ bsize: BlockSize, bo: &BlockOffset) -> f64 {
     }
 
     let bs = mi_size_wide[bsize as usize];
-    let is_too_big =
-        bo.x as i32 + bs as i32 - fi.w_in_b as i32 > 0 ||
-        bo.y as i32 + bs as i32 - fi.h_in_b as i32 > 0;
+    let must_split = bo.x + bs as usize > fi.w_in_b || bo.y + bs as usize > fi.h_in_b;
 
     let is_splittable = bsize > fi.min_partition_size;
-    let is_codable = bsize < BlockSize::BLOCK_64X64 && !is_too_big;
+    let is_codable_unsplit = bsize < BlockSize::BLOCK_64X64 && !must_split;
```

Comment:
Is is_codable_unsplit the exact opposite of must_split? It would be nice to be consistent here if you're renaming.

Reply:
No, it just means that the partition is codable as a leaf node (e.g. as `PARTITION_NONE`).